src/components/DisplayInput.js

```diff
@@ -0,0 +1,145 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const DisplayInput = React.createClass({
+  propTypes: {
+    hint: React.PropTypes.string,
+    label: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    primaryColor: React.PropTypes.string,
+    showHint: React.PropTypes.bool,
+    status: React.PropTypes.shape({
+      type: React.PropTypes.string,
+      message: React.PropTypes.string
+    }),
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div style={styles.wrapper}>
+        {this.props.label ?
+          <div key='label' style={styles.label}>
+            <div style={styles.labelText}>{this.props.label}</div>
+          </div> : null
+        }
+        <input
+          {...this.props}
+          key='input'
+          label={this.props.label}
+          style={styles.input}
+          type='text'
+        />
+        <div style={styles.hint}>
+          <div style={styles.hintText}>
+            {this.props.showHint && !this.props.status ? <div>{this.props.hint}</div> : null}
+            {this.props.status ? <div style={styles[this.props.status.type]}>{this.props.status.message}</div> : null}
+          </div>
+        </div>
+      </div>
+    );
+  },
+
+  styles () {
+    return {
+      wrapper: Object.assign({
+        borderBottom: this.props.valid ? '1px solid ' + StyleConstants.Colors.ASH : '1px solid ' + StyleConstants.Colors.STRAWBERRY,
+        borderLeft: '1px solid transparent',
+        borderRadius: 3,
+        borderRight: '1px solid transparent',
+        borderTop: '1px solid transparent',
```

Comment:
I'm not an css expert so some one correct me if I'm wrong here, but specifying the borders for left, right, and top is not required here right?  By omitting them they should end up as none I would think.